Examples/Tests/flux_injection/inputs_3d

```diff
@@ -38,8 +38,8 @@ electron.num_particles_per_cell = 100
 electron.surface_flux_pos = -4.
 electron.flux_normal_axis = y
 electron.flux_direction = +1
-electron.profile = constant
-electron.density = 1.
+electron.flux_profile = parse_flux_function
+electron.flux_function(x,y,z,t) = "1."
```

Comment:
Even though the flux is constant in this example, I did use the `parse_flux_function` here just to test the corresponding code path.